swarm_rl/models/quad_multi_model.py

```diff
@@ -169,11 +169,14 @@ def __init__(self, cfg, obs_space):
         # Attention Layer
         self.attention_layer = MultiHeadAttention(4, cfg.rnn_size, cfg.rnn_size, cfg.rnn_size)
 
+        # # This is for sim2real
+        # self.attention_layer = OneHeadAttention(cfg.rnn_size)
+
         # MLP Layer
-        self.feed_forward = nn.Sequential(fc_layer(3 * cfg.rnn_size, 2 * cfg.rnn_size),
+        self.feed_forward = nn.Sequential(fc_layer(3 * cfg.rnn_size, cfg.rnn_size),
```

Comment:
If we change the final feed_forward layer from 2 * cfg.rnn_size to cfg.rnn_size, will that harm the final performance? If the answer is not, can you show me the wandb report?

Reply:
This is just for sim2real. In simulation it's still 2 * cfg.rnn_size

Reply:
If this is for sim2real, then, you probably need a hyperparameter to decide sim2real or just for sim.